docker-storage-setup.sh

```diff
@@ -657,26 +657,36 @@ EOF
     Fatal "udevadm settle after partition creation failed. Exiting."
   fi
 
-  if ! wait_for_dev ${dev}1; then
-    Fatal "Partition device ${dev}1 is not available"
+  if ! wait_for_dev ${part}; then
+    Fatal "Partition device ${part} is not available"
   fi
 }
 
+partition_name() {
+    if echo "$1" | grep -q '[0-9]$'; then
+        echo "${1}p1"
+    else
+        echo "${1}1"
+    fi
+}
```

Comment:
How about if we scan the name of partition from disk using "lsblk" command. I think "lsblk -l" lists devices and all its partitions and we can scan that to figure out name of the partition.